modules/47_regipol/regiCarbonPrice/bounds.gms

```diff
@@ -42,8 +42,10 @@ $ENDIF.CCSinvestment
 
 
 ** Force historical bounds on coal
-    vm_cap.up("2020",regi,"pc","1")$((cm_startyear le 2020) and (sameas(regi,"DEU"))) = 38.028/1000;
-    vm_cap.up("2020",regi,"pc","1")$((cm_startyear le 2020) and (sameas(regi,"UKI"))) = 1.3/1000; !!2019 capacity = 7TWh, capacity factor = 0.6 ->  ~1.35GW -> Assuming no new capacity -> average 2018-2022 = ~ 1GW
+vm_cap.up("2020",regi,"pc","1")$((cm_startyear le 2020) and (sameas(regi,"DEU"))) = 38.028/1000;
+*** 2019 capacity = 7TWh, capacity factor = 0.6 ->  ~1.35GW -> Assuming no new capacity -> average 2018-2022 = ~ 1GW
+*** Commented out as this line causes issues for H12 regionmappings
```

Comment:
I just saw this. I don't get why this would cause issues with H12 as if that was the case, the DEU constraint would also cause issues.

Reply:
Ye, this causes issues with H12 regipol as I said. Can you please fix this? I would commit it now such that it works also for others.

Reply:
Again, I don't see any reason why this would cause any issues with H12. 
The region specific coding is inside a sameas conditional. If this would cause issues, we would have a lot of more issues everywhere in the code.
Sorry, but I don't see anything to be fixed here.